libs/auth/react/src/lib/Authenticator/AuthenticatorLoadingScreen.tsx

```diff
@@ -1,14 +1,16 @@
-import React, { FC } from 'react'
+import React from 'react'
 import { Box, LoadingIcon } from '@island.is/island-ui/core'
-import * as styles from './Authenticator.treat'
+import * as styles from './AuthenticatorLoadingScreen.treat'
 
-const AuthenticatorLoadingScreen: FC<{}> = () => {
+const AuthenticatorLoadingScreen = () => {
   return (
     <Box
       display="flex"
       justifyContent="center"
       alignItems="center"
       className={styles.wrapper}
+      role="progressbar"
+      aria-valuetext="Er að vinna í innskráningu"
```

Comment:
Translation for that copy?

Reply:
That's tricky :/ 

We haven't loaded any translations at this point, while loading/authenticating.